colossus-metrics/src/main/scala/colossus/metrics/collectors/Histogram.scala

```diff
@@ -43,7 +45,54 @@ object Histogram {
     }.toVector
     BucketList(buckets)
   }
-  
+
+  /**
+    * Create a Histogram with the following address.  Note, the address will be prefixed by the MetricSystem's root.
+    * Configuration is resolved and overlayed as follows('metricSystemConfigPath' is the configPath parameter, if any, that was
+    * passed into the MetricSystem.apply function):
+    * 1) metricSystemConfigPath.address
+    * 2) metricSystemConfigPath.collectors-defaults.histogram
+    * 3) colossus.metrics.collectors-defaults.histogram
+    * @param address The address relative to the Collection's MetricSystem Root.
+    * @param collection The Collection this Metric will become a part of.
+    * @return Created Histogram.
+    */
+  def apply(address : MetricAddress)(implicit collection : Collection) : Histogram = {
+
+    import scala.collection.JavaConversions._
+
+    val params = resolveConfig(collection.config.config, s"colossus.metrics.$address", DefaultConfigPath)
+    val percentiles = params.getDoubleList("percentiles").map(_.toDouble)
+    val sampleRate = params.getDouble("sample-rate")
+    val pruneEmpty = params.getBoolean("prune-empty")
+    apply(address, percentiles, sampleRate, pruneEmpty)
+  }
+
+  /**
+    * Create a Histogram with following address.  Note, the address will be prefixed by the MetricSystem's root.
+    * Configuration is resolved and overlayed as follows('metricSystemConfigPath' is the configPath parameter, if any, that was
+    * passed into the MetricSystem.apply function):
+    * 1) configPath.address
+    * 2) metricSystemConfigPath.collectors-defaults.histogram
+    * 3) colossus.metrics.collectors-defaults.histogram
+    * @param address The address relative to the Collection's MetricSystem Root.
+    * @param configPath The path in the ConfigFile that this Histogram is located.
+    * @param collection The Collection this Metric will become a part of.
+    * @return
+    */
+  def apply(address : MetricAddress, configPath : String)(implicit collection : Collection) : Histogram = {
```

Comment:
Can't `configPath` have a default argument of "colossus.metrics" and then you can eliminate the other constructor?